diff --git a/tests/myth_aligned_alloc.c b/tests/myth_aligned_alloc.c
index 84bde17..988cd5d 100644
--- a/tests/myth_aligned_alloc.c
+++ b/tests/myth_aligned_alloc.c
@@ -10,7 +10,7 @@ void * aligned_alloc(size_t al, size_t sz);
 
 int main(int argc, char ** argv) {
   size_t al = (argc > 1 ? atol(argv[1]) : 32);
-  size_t sz = (argc > 2 ? atol(argv[2]) : 35);
+  size_t sz = (argc > 2 ? atol(argv[2]) : 64);
   size_t n  = (argc > 3 ? atol(argv[3]) : 3);
   size_t i;
   for (i = 0; i < n; i++) {
